# DP: Traditional GNU systems don't have a /usr directory.  However, Debian
# DP: systems do, and we support both having a /usr -> . symlink, and having a
# DP: /usr directory like the other ports.  So this patch should NOT go
# DP: upstream.

---
 gcc/config/gnu.h |    3 ++-
 gcc/config/t-gnu |    2 +-
 2 files changed, 3 insertions(+), 2 deletions(-)

--- a/src/gcc/config/gnu.h
+++ b/src/gcc/config/gnu.h
@@ -30,7 +30,8 @@ along with GCC.  If not, see <http://www.gnu.org/licenses/>.
 
 /* Standard include directory.  In GNU, "/usr" is a four-letter word.  */
 #undef STANDARD_INCLUDE_DIR
-#define STANDARD_INCLUDE_DIR "/include"
+/* Overriden for Debian GNU/Hurd.  */
+#define STANDARD_INCLUDE_DIR "/usr/include"
 
 #undef LINUX_TARGET_OS_CPP_BUILTINS
 #define LINUX_TARGET_OS_CPP_BUILTINS()		\
--- a/src/gcc/config/t-gnu
+++ b/src/gcc/config/t-gnu
@@ -1,2 +1,2 @@
 # In GNU, "/usr" is a four-letter word.
-NATIVE_SYSTEM_HEADER_DIR = /include
+NATIVE_SYSTEM_HEADER_DIR = /usr/include
